README.md

```diff
@@ -88,7 +87,30 @@ asreview semantic_clustering -f van_de_Schoot_2017.csv -o output.csv
 
 If an output file is not specified, `output.csv` is used as output file name.
 
-### Dashboard
+### Test file
+```shell
+asreview semantic_clustering -t -o <output_file.csv>
+```
+
+Using `-t` instead of `-f` uses the
+[`van_de_Schoot_2017`](https://asreview.readthedocs.io/en/latest/intro/datasets.html#featured-datasets)
+dataset as input file. This way, the plugin can easily be tested.
+
+### Transformer
+Semantic Clustering uses the
+[`allenai/scibert_scivocab_uncased`](https://github.com/allenai/scibert)
+transformer model as default setting. Using the `--transformer <model>` option,
+another model can be selected for use instead:
+
+```shell
+asreview semantic_clustering -t -o <output_file.csv> --transformer bert-base-uncased
+```
+
+Any pretrained model will work.
+[Here](https://huggingface.co/transformers/pretrained_models.html) is an example
```

Comment:
In ASReview documentation, we try to follow the Google documentation style guide. Please see this item: https://developers.google.com/style/cross-references#references-to-other-documents